esp8266-weather-station-color.ino

```diff
@@ -848,6 +851,7 @@ uint8_t changeScreen(TS_Point p, uint8_t screen) {
   // if (p.x <= dividerMiddle)  Serial.print(" right ");  // <= 120
   // Serial.println();
 
+#ifndef ROTATE_180
```

Comment:
I personally don't like inverted if/else conditions. Can you start with the truthy condition (`ifdef`)?